docs/technical_specs/model_libraries/power_generation/unit_models/boiler_heat_exchanger_3streams.rst

```diff
@@ -0,0 +1,91 @@
+Heat Exchanger With Three Streams
+=================================
+
+.. index::
+   pair: idaes.power_generation.unit_models.heat_exchanger_3streams; HeatExchangerWith3Streams
+
+.. currentmodule:: idaes.power_generation.unit_models.heat_exchanger_3streams
+
+The HeatExchangerWith3Streams model consists of a heat exchanger with three inlets, `side_1` represents the hot stream, while `side_2` and `side_3` are cold streams.
+This model is a simplyfied generic heat exchanger model with lumped UA (the product of the overall heat transfer coefficient and the heat transfer area).
+
+In a power plant flowsheet this model is used to represent an air preheater unit. This is because modeling the Ljungström type preheater is quite challenging since it involves not only the hot and cold gas streams but also the energy stored in and relased from the metal parts. 
+
+
+Degrees of Freedom
+------------------
+
+Aside from the inlet conditions, a heat exchanger 3 inlets model usually has six degrees
+of freedom, which can be fixed for it to be fully specified. Things that are
+frequently fixed are two of:
+
+* UA_side_2 - lumped overall heat transfer and heat transfer area of side 2
+* UA_side_3 - lumped overall heat transfer and heat transfer area of side 3
+* frac_heatloss - fraction of heat loss in the system
+* deltaP_side_1 - pressure drop in side 1
+* deltaP_side_2 - pressure drop in side 2
+* deltaP_side_3 - pressure drop in side 3
+
+
+Model Structure
+---------------
+
+The ``HeatExchangerWith3Streams`` model contains three ``ControlVolume0DBlock`` blocks. By default the
+hot side is named ``side_1`` and two cold sides are named ``side_2 and side_3``. These names are not configurable.
```

Comment:
Typo: ``side_2 and side_3`` should be marked separately, i.e. ``side_2`` and ``side_3``.

Also, seeing as the names are not configurable in this case, "By default" should be removed from the beginning of this sentence.